conf/base.config

```diff
@@ -63,6 +63,21 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
+    withName:MINIMAP2_ALIGN_VARIANT{
+        ext.args   = { "--MD -ax map-ont --eqx" }
+    }
+    withName:SAMTOOLS_VIEW {
+        ext.args = '--output-fmt bam'
+    }
+    withName:BEDTOOLS_GENOMECOV {
+        ext.args = '-bg'
+    }
+    withName:SUBREAD_FEATURECOUNTS_GENE {
+        ext.args = '-g gene_id -t exon -o counts_gene.txt'
+    }
+    withName:SUBREAD_FEATURECOUNTS_TRANSCRIPT{
+        ext.args = '--primary --fraction -F GTF -g transcript_id -t transcript --extraAttributes gene_id -o counts_transcript.txt'
+    }
```

Comment:
These should be in the `conf/modules.config` file

Reply:
I moved these to `conf/modules.config`